### PR TITLE
Todo-List App

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**

### CODE REVIEW

File: react/react1/week1/todo_list/src/App.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function TodoItem(props) {
+  return (
+    <div>
+      <h3>{props.description}</h3>
+      <p>{props.deadline}</p>
+    </div>
+  );
+}
```
Comment (YurijPaharev) [inline react/react1/week1/todo_list/src/App.js:10]: Please put components to separate files (TodoItem.js)

File: react/react1/week1/todo_list/src/App.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    {
+      description: 'Brush teeth',
+      deadline: 'Thu Sep 14 2017'
+    },
+    {
+      description: 'Eat breakfast',
+      deadline: 'Fri Sep 15 2017'
+    }
+  ];
```
Comment (YurijPaharev) [inline react/react1/week1/todo_list/src/App.js:26]: You should probably want to move it to a separate file

**Review by YurijPaharev — COMMENTED** [review]
Nice work, just a couple comments for you

File: react/react1/week1/todo_list/src/App.js (hunk 1 above)
Comment (Atefeh-Behroozi) [reply react/react1/week1/todo_list/src/App.js:10]: Ok, sure. I'll do it now. Thank you

File: react/react1/week1/todo_list/src/App.js (hunk 2 above)
Comment (Atefeh-Behroozi) [reply react/react1/week1/todo_list/src/App.js:26]: Ok, I add new file, the name is TodoList.js and move these codes there. Thanks

**Comment by Atefeh-Behroozi** [comment]
Thank you for the feedback. I've added two new files and put codes in separate files.

**Pull request merged**